ospfd/ospf_vty.c

```diff
@@ -9779,7 +9779,7 @@ static int config_write_interface_one(struct vty *vty, struct vrf *vrf)
 			}
 
 			/* bfd  print. */
-			if (params->bfd_info)
+			if (params && params->bfd_info)
```

Comment:
I think we should use `OSPF_IF_PARAM_CONFIGURED` macro here to keep the code pattern in case we ever change that verification.

Reply:
@rzalamena bdf_info is not defined using "DECLARE_IF_PARAM". Do you want me to change that into ospfd/ospf_interface.h?

Reply:
Sorry about this, after a more careful look I think it is best to keep as is.

Using `DECLARE_IF_PARAM` will generate a new `uint8_t` that is not going to be used.